webhooks-extension/development_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Set your docker registry and install namespace here so you do not have to
+# enter them every time you run this script
+DOCKER_REPO=""
+NAMESPACE=""
+
+if [ -z $DOCKER_REPO ]; then
+    read -p "Enter your docker registry: " DOCKER_REPO
+fi
+if [ -z $NAMESPACE ]; then
+    read -p "Enter your install namespace: " NAMESPACE
+fi
+echo "DOCKER_REPO: $DOCKER_REPO"
+echo "NAMESPACE: $NAMESPACE"
+
+
+# Build and push the extension and sink images
+docker login
+
+echo "build cmd/extension ..."
+docker build -t ${DOCKER_REPO}/extension:latest -f cmd/extension/Dockerfile .
+docker push ${DOCKER_REPO}/extension:latest
+
+echo "build cmd/sink ..."
+docker build -t ${DOCKER_REPO}/extension-sink:latest -f cmd/sink/Dockerfile .
+docker push ${DOCKER_REPO}/extension-sink:latest
+
+
+# Copy and replace the config/ yaml files into install/
+mkdir -p install
+cp -r config/ install/
+for file in install/*.yaml; do
+    sed -i '' 's/DOCKER_REPO/'"$DOCKER_REPO"'/g' "${file}"
+done
+
+
+# Install into the NAMESPACE
+echo "instal in namespace ${NAMESPACE}"
+kubectl apply -f install/ -n ${NAMESPACE}
```

Comment:
Include a message as to what to do next please, a user may be expecting pods. 

I get a ksvc in the install namespace that waits for its revision to be ready. The revision takes over nine minutes to become ready in my case, so a user would be confused thinking "what's going on!" potentially.

```
webhooks-extension-7754bc5584-8w7cw                             1/1       Running     0          7m13s
webhooks-extension-sink-zkmft-deployment-64b5c8b447-ngtp8       1/3       Running     1          7m13s
```

Then the probes fail

```
  Warning  Unhealthy  7m                 kubelet, docker-desktop  Liveness probe failed: Get http://10.1.0.66:8012/liveness: dial tcp 10.1.0.66:8012: connect: connection refused
  Warning  Unhealthy  7m                 kubelet, docker-desktop  Readiness probe failed: Get http://10.1.0.66:8012/readiness: dial tcp 10.1.0.66:8012: connect: connection refused
  Warning  Unhealthy  2m (x296 over 7m)  kubelet, docker-desktop  Readiness probe failed: Get http://10.1.0.66:8022/health: dial tcp 10.1.0.66:8022: connect: connection refused
```

Sorry...giving up at this stage!

Reply:
Actually

```
adams-mbp:webhooks-extension aroberts$ kgp
NAME                                                            READY     STATUS      RESTARTS   AGE
tekton-dashboard-787b9df4db-zc55b                               1/1       Running     5          28h
webhooks-extension-7754bc5584-8w7cw                             1/1       Running     1          6h20m
```

I had dead istio pilot pods and this explains why my ksvc was never ready! Only quality of life improvements from me please 😄